build.gradle

```diff
@@ -79,6 +84,8 @@ allprojects {
         withSourcesJar()
         withJavadocJar()
     }
+
+    project.getExtensions().getExtraProperties().set("versions", VersionProperties.getVersions())
```

Comment:
Thanks for doing this!  It was on my TODO list to figure out how to integrate the new version catalog.